sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -340,6 +340,20 @@ certificate will be added with the alias of `mycert`.
     4. Certificates from Azure Key Vault. 
     5. Certificates from classpath.
 
+### Key-Less certificates
+You can set the private key as [non-exportable] to ensure the security of the key.
+
```

Comment:
Note if you want to use ~a~(delete `a`) key less certificate, you must add `sign` permission~s~(delete `s`).

You can add permission in portal: ...

Or add permission by cli command: ...